cv32e40s/bsp/crt0.S

```diff
@@ -1,5 +1,6 @@
 /* Copyright (c) 2017  SiFive Inc. All rights reserved.
  * Copyright (c) 2019  ETH Zürich and University of Bologna
+ * Copyright (c) 2022  Silicon Laboratories Inc.
```

Comment:
Why specifically 2022? The file was added to the 40s dir in 2021, and today's current year is 2023.

Reply:
Good catch - probably typo or modified last year (this has been on a stale dev branch for some time)

Reply:
> or modified last year (this has been on a stale dev branch for some time)

If it was actually written in 2022 then I think the copyright should be `yyyy <= 2022`. (But who really knows for sure..?)